talend-component-maven-plugin/src/it/web/.jenkins/scripts/tcomp-build.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+#
+#  Copyright (C) 2006-2022 Talend Inc. - www.talend.com
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+# set -xe
+
+# Quick tcomp build without any test and facultative modules (documentation...)
+# Parameters
+[ -z ${1+x} ] && usage "Parameter 'pom_file_path'"
+
+pom_file_path=${1}
+
+# Constants
+MAVEN_FAST_INSTALL_CMD="mvn clean install \
+                -Dspotless.apply.skip=true \
+                -Dspotbugs.skip=true \
+                -Dcheckstyle.skip=true -Drat.skip=true \
+                -DskipTests \
+                -Dmaven.javadoc.skip=true \
+                -Dinvoker.skip=true \
+                -Dmaven.artifact.threads=25"
+
```

Comment:
Please use lower case variables inside the script to use local variables instead of environment variables. Unless it is required ?
Same remark for other variables used afterwards.

Reply:
I used UPPER for global and constant
Constant will always start with "_"